src/metabase/events/schema.clj

```diff
@@ -1,48 +1,54 @@
 (ns metabase.events.schema
   (:require
-   [malli.core :as mc]
-   [malli.generator :as mg]
+   [malli.json-schema :as mjs]
    [malli.util :as mut]
-   [metabase.lib.schema.common :as common]
+   [metabase.api.macros.defendpoint.open-api :as defendpoint.open-api]
    [metabase.models.view-log-impl :as view-log-impl]
-   [metabase.util.malli :as mu]
    [metabase.util.malli.registry :as mr]
    [metabase.util.malli.schema :as ms]
-   [nano-id.core :as nano-id]
    [toucan2.core :as t2]))
 
-(mu/defn event-schema
+(defmulti event-schema
   "Get the Malli schema we should use for events of `topic`. By default, this looks in our registry for a schema
   matching the event topic name; if it fails to find one, it falls back to `:map`."
-  [topic :- keyword?]
+  {:arglists '([topic event-info])}
+  (fn [topic _event-info] topic))
```

Comment:
Just out of interest, how do you imagine we'll start using `_event-info` here?

I guess one nice thing we could do is add empirical entries for everything in this map, in the fallback.